pkg/client/outofband/models.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package outofband
+
+import "github.com/hyperledger/aries-framework-go/pkg/didcomm/protocol/decorator"
+
+// Request embeds the sender's request.
+type Request struct {
```

Comment:
Aren's we going to use `"protocols"`?

Reply:
https://github.com/hyperledger/aries-rfcs/blob/master/features/0434-outofband/README.md#message-type-httpsdidcommorgoob-invitationverinvitation

Reply:
`invitation` will be added in a separate PR. This PR right now has 600 lines.

Reply:
looks good